packages/server/src/utilities/rowProcessor/bbReferenceProcessor.ts

```diff
@@ -6,20 +6,25 @@ import { InvalidBBRefError } from "./errors"
 export async function processInputBBReferences(
   value: string | string[] | { _id: string } | { _id: string }[],
   subtype: FieldSubtype
-): Promise<string> {
-  const result: string[] = []
+): Promise<string | undefined> {
+  const referenceIds: string[] = []
+
+  if (Array.isArray(value)) {
+    referenceIds.push(...value.map(x => (typeof x === "string" ? x : x._id)))
```

Comment:
can we use `id` or `possibleObj` rather than `x`? When looking at a lot of different array functions I do mind it a lot more readable to see something contextual.